certbot/compat/os.py

```diff
@@ -29,3 +29,18 @@
 
 # Clean all remaining importables that are not from the core os module.
 del ourselves, std_os, std_sys
+
+
+# Because of the blocking strategy on file handlers on Windows, rename do not behave as expected
+# with POSIX systems: an exception will be raised if dst already exists.
+def rename(*unused_args, **unused_kwargs):  # pylint: disable=function-redefined
```

Comment:
nit: Given the purpose of this module, I'd personally just move a single `pylint: disable=function-redefined` above these function definitions at the top level of the file.

Reply:
Done.